db/migrate/20210223062251_change_index_from_name_to_email.rb

```diff
@@ -0,0 +1,9 @@
+class AddUniquenessEmailColumnInUsers < ActiveRecord::Migration[6.0]
```

Comment:
At the time of migrations it shows this error:

![imagen](https://user-images.githubusercontent.com/12243331/108875108-727f9080-75c2-11eb-95a0-374bc741ab2b.png)